db/migrate/20240219082737_add_registration_start_date_to_cohorts.rb

```diff
@@ -0,0 +1,5 @@
+class AddRegistrationStartDateToCohorts < ActiveRecord::Migration[7.1]
+  def change
+    add_column :cohorts, :registration_start_date, :datetime, null: false
```

Comment:
could we include this in the same migration please as this pr is introducing the table?

Reply:
Yeah I agree, I tend to `db:rollback` and `db:migrate` when working on a branch so there are fewer separate migrations per PR